package.json

```diff
@@ -157,6 +157,7 @@
         {
           "files": "./packages/vx-demo/**",
           "rules": {
+            "@typescript-eslint/no-explicit-any": "warn",
```

Comment:
`"@typescript-eslint/no-explicit-any": ["warn", { "fixToUnknown": false }]`

Add `fixToUnknown: false` to avoid `eslint` auto-fix replacing `any` to `unknown` on file save